components/definitions/newTab.d.ts

```diff
@@ -39,14 +39,19 @@ declare namespace NewTab {
     destinationUrl: string
   }
 
+  export type BrandedWallpaperType = 'image' | 'richMedia'
```

Comment:
Hey not your fault because everything else is in here but we're trying to avoid adding everything to a global types file.

Instead, create them near we're they're used - in this case you'll still need to import `BrandedWallpaperType` into this file.

Reply:
Moved to appropriate place, thanks!